social_protection/schema.py

```diff
@@ -157,6 +158,11 @@ def resolve_benefit_plan(self, info, **kwargs):
         if beneficiary_status:
             filters.append(Q(beneficiary__status=beneficiary_status) | Q(groupbeneficiary__status=beneficiary_status))
 
+        search = kwargs.get("search", None)
```

Comment:
Why we define addtional 'search' filter? I think we can use directly from default graphQL schema for BenefitPlan. (definied by default name and code)

Reply:
You are right, I was doing it analogically to the Product picker and didn't put much thought into that.

Reply:
In that case please fix your solution in fe modules to use filters provided by graphQL schema by default (from ORM)